docs/pages/machine-id/guides/applications.mdx

```diff
@@ -0,0 +1,281 @@
+---
+title: Machine ID with Application Access
+description: Accessing web applications with Machine ID
+---
+
+In this guide, we will demonstrate how to configure an automated service to use Machine ID to access a web application protected by Teleport.
+
+With Machine ID, Teleport issues short-lived certificates, tied to a machine
+identity, that can be rotated, audited, and managed with the same access
+controls that Teleport provides for human users.
+
+## Prerequisites
+
+<ScopedBlock scope={["cloud"]}>
+You will need Teleport Cloud 10.1.0 or later.
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+You will need to be running Teleport version 10.1.0 or later.
```

Comment:
```suggestion
You will need a running Teleport cluster version 10.1.0 or later.
```
To distinguish running a cluster from, say, having a single `teleport` binary running on a host.